src/install-pnpm/run.ts

```diff
@@ -95,4 +104,21 @@ Please specify it by one of the following ways:
   return packageManager
 }
 
+function getPnpmVersionFromLockfile(
+  lockfileVersion: string | undefined
+): string | undefined {
+  switch (true) {
+    case lockfileVersion === '5.3':
+      return '6';
+    case lockfileVersion === '5.4':
+      return '7';
+    case lockfileVersion === '6.0' || lockfileVersion === '6.1':
+      return '8';
+    case lockfileVersion === '7.0' || lockfileVersion === '9.0':
+      return '9';
```

Comment:
could we use dist tags instead? Like `latest-9`, `latest-8`, etc.

These dist-tags exist.

Reply:
done